frontend/packages/dev-console/src/components/import/builder/BuilderImageTagSelector.tsx

```diff
@@ -62,14 +68,16 @@ const BuilderImageTagSelector: React.FC<BuilderImageTagSelectorProps> = ({
 
   return (
     <>
-      <DropdownField
-        name="image.tag"
-        label={t('devconsole~Builder Image version')}
-        items={tagItems}
-        title={tagItems[selectedImageTag]}
-        fullWidth
-        required
-      />
+      <div style={!showEditImportStrategy ? { display: 'none' } : {}}>
```

Comment:
Why do you need this check here? You should not render the whole `BuildImageSection` component when the strategy is not builder image.